### PR TITLE
Feat/modal and toast

### PR DESCRIPTION
**Pull request by tonymike99**
I've added modal and toast components.

### CODE REVIEW

File: Components/Modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+.modal-visible {
+    background-color: var(--light-color);
+    opacity: 1;
+    visibility: visible;
+    z-index: 1;
+}
+
+.modal-overlay {
+    background-color: rgba(0, 0, 0, 0.2);
```
Comment (iRohitGaur) [inline Components/Modal/modal.css:23]: use variable

**Review by iRohitGaur — APPROVED** [review]
Good to merge 👍🏻

File: Components/Modal/modal.css (hunk 1 above)
Comment (tonymike99) [reply Components/Modal/modal.css:23]: Sure 👍

**Review by purohitdheeraj — APPROVED** [review]
Good to merge 
loved the hover gradient it looks awesome

**Pull request merged**